--- origsrc/mod_perl-1.30/apaci/Makefile.tmpl	2007-03-30 00:12:45.000000000 -0500
+++ src/mod_perl-1.30/apaci/Makefile.tmpl	2008-01-07 21:22:57.562500000 -0600
@@ -56,7 +56,7 @@
 	$(MP_AR) crv $@ $(MP_OBJS)
 	$(MP_RANLIB) $@
 
-libperl.so: $(MP_OBJS_PIC)
+libperl.dll: $(MP_OBJS_PIC)
 	$(MP_RM) $@
 	$(MP_LD) $(MP_LDFLAGS_SHLIB) -o $@ $(MP_OBJS_PIC) $(MP_LIBS)
 
@@ -94,7 +94,8 @@
 
 #   FOR STAND-ALONE BUILDING ONLY
 install:
-	$(APXS) -i -a -n perl libperl.so
+	mkdir -p $(DESTDIR)$(APACHELIBEXEC)
+	install libperl.$(LIBEXT) $(DESTDIR)$(APACHELIBEXEC)
 
 depend:
 	cp Makefile.tmpl Makefile.tmpl.bak \
